content/docs/0.6.0/installation/backup_and_restore/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Backup and restore Keptn
+description: How to backup and restore Keptn.
+weight: 20
+keywords: backup
+---
+
+# Backup Keptn 
+
+To secure all your data in your project's git repositories, as well as all events that have occurred for these project, you need to 
+back up the data within the Configuration Service, the MongoDB and - if you have configured those - credentials to your project's git upstream repos.
+The following sections describe how to back up that data and store it on your local machine.
+**NOTE:** These instructions do not cover backing up any Helm- or Istio-related configurations.
+
+## Backup Configuration Service
+
+1. First, create a local backup directory containing all data within the git repos of the configuration-service. To do so, please execute the following command:
+
+```console
+mkdir keptn-backup
+cd keptn-backup
+mkdir config-svc-backup
+CONFIG_SERVICE_POD=$(kubectl get pods -n keptn -lrun=configuration-service -ojsonpath='{.items[0].metadata.name}')
+kubectl cp keptn/$CONFIG_SERVICE_POD:/data ./config-svc-backup/ -c configuration-service
+```
+
+1. Verify that the data has been copied correctly to your local machine by checking the content of the directory you just created. 
+This should include all projects you have onboarded with Keptn, as well as a `lost+found` directory, which can be ignored.
+
+```console
+$ ls config-svc-backup/config
+lost+found	sockshop my-project
+```
+
+## Backup Mongodb Data
+
+The next step is to create a backup of the mongodb database in the `keptn-datastore` namespace using the `mongodump` tool.
+NOTE: this tool will be executed within the pod running the MongoDB, so you do not need to have it installed locally.
+
+1. Create a directory for the MongoDB backup:
+
+```console
+mkdir mongodb-backup
+chmod o+w mongodb-backup
+```
+
+1. Copy the data from the MongoDB to your local directory:
+
+```console
+MONGODB_USER=$(kubectl get secret mongodb-credentials -n keptn -ojsonpath={.data.user} | base64 -d)
```

Comment:
The mongodb_user and password is actually not needed here. Right?

Reply:
correct, in this case it's not needed. I have removed it